tests/compile-fail/modulo_one.rs

```diff
@@ -0,0 +1,7 @@
+#![feature(plugin)]
+#![plugin(clippy)]
+#![deny(modulo_one)]
+
+fn main() {
```

Comment:
There should be some passing tests here too.

(AST representation changes often so stuff like this could magically stop working or start catching too much)